File: Selenium/Activities/Activity1.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net");
        String  ele = driver.getTitle();
        System.out.println("Title is : "+ ele);
        driver.findElement(By.id("about-link")).click();
        System.out.println("Tile of about us : " + driver.getTitle());
        driver.quit();
    }
}

File: Selenium/Activities/Activity1.py
from webdriver_manager.firefox import GeckoDriverManager
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net")
    tit = driver.title
    print(tit)
    driver.find_element(By.ID, "about-link").click()
    new_tit = driver.title
    print(new_tit)
driver.quit()

